Nodejs/Product/Nodejs/SharedProject/CommonFolderNode.cs

```diff
@@ -227,15 +227,7 @@ internal override int IncludeInProjectWithProgress(bool includeChildren)
         public override void RenameFolder(string newName)
         {
             var oldName = this.Url;
-            this._project.SuppressFileChangeNotifications();
-            try
-            {
-                base.RenameFolder(newName);
```

Comment:
>base.RenameFolder(newName); [](start = 16, length = 27)

Does the new watcher no longer report renames?  Why is suppression no longer required?

Reply:
We only get one notificiation in the new watcher for the older, vs. a notification for each file in the old watcher. Further the new watcher doesn't allow suppressing notifications on subfolders specifically, so we would need to stop notifications on all folders, and I don't want to miss notifications on those other folders while renaming.